src/components/FriendSelector.tsx

```diff
@@ -45,12 +45,14 @@ const FriendSelectorBoxText = styled.p`
 `;
 
 function FriendSelector() {
+  const userId = sessionStorage.getItem('userId')?.split('@')[0] ?? '';
```

Comment:
만약 서비스 전반에서 userId와 같이 인증 정보를 사용하는 일이 많아진다면, 매번 컴포넌트에서 세션 스토리지에 접근하는 코드를 작성해야 할 것 같은데요, 보다 나은 개발 경험과 유지보수성 + 가독성까지 챙길 수 있도록 인증 정보를 관리하고 반환해주는 커스텀 훅을 만들면 좋을 것 같습니다!

`userId`라는 리액트 state를 반환하는 `useUser` 커스텀 훅을 구현해볼 수 있을까요?

```suggestion
  const { userId } = useUser();
```